pkg/utils/resources/resources.go

```diff
@@ -146,6 +146,25 @@ func Cmp(lhs resource.Quantity, rhs resource.Quantity) int {
 	return lhs.Cmp(rhs)
 }
 
+func Eql(lhs v1.ResourceList, rhs v1.ResourceList) bool {
```

Comment:
Two characters for less terse sounds like a nice tradeoff to me :)

```suggestion
func Equal(lhs v1.ResourceList, rhs v1.ResourceList) bool {
```